data/sql/updates/pending_db_world/rev_1584389363968563900.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1584389363968563900');
+
+UPDATE `quest_offer_reward` SET `RewardText`= 'This... this pendant.  I gave it to Old Whitebark after his people helped us rebuild our village.$B$BI guess this means he\'s...$B$B<The blood elf clears her throat as she regains her composure.>$B$BI appreciate you bringing this to me, $N.  There is something I\'d like to ask of you.' WHERE `ID`=8474;
```

Comment:
use two single quotes instead of  backslash

`it''s perfect`

`it\'s ok but not perfect`